packages/wallets/src/features/cfd/modals/CTraderSuccessModal/CTraderSuccessModal.tsx

```diff
@@ -15,32 +15,40 @@ type TCTraderSuccessModal = {
 
 const CTraderSuccessModal = ({ isDemo, walletCurrencyType }: TCTraderSuccessModal) => {
     const { data: cTraderAccounts } = useCtraderAccountsList();
+    const { data: activeWallet } = useActiveWalletAccount();
```

Comment:
@lubega-deriv 
since we are already passing props from active wallet, could we pass display_name too? and remove `useActiveWalletAccount` from here?

Reply:
Updated